### PR TITLE
feature/build-with-wepack

### PR DESCRIPTION
**Pull request by cdaringe**
#### asana

https://app.asana.com/0/10663154302695/67550252004702
# problem statement
- future plans include exploiting webpack plugins, namely, the `DefinePlugin`
# solution
- change build to use webpack
- update modules to be CommonJS, but export UMD compatible package

no package business logic intended changes were made in the PR (or at least intended to be :) )
# review

recommend reviewing with: https://github.com/MRN-Code/coins-logon-widget/pull/3/files?w=1

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -2,139 +2,17 @@
 'use strict';
 
 module.exports = function(grunt) {
-    require('load-grunt-tasks')(grunt);
-
     grunt.initConfig({
-        config: {
```
Comment (cdaringe) [inline Gruntfile.js:8]: i pulled these out to follow the node_boilerplate. hope thats cool

File: grunt/clean.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+'use strict';
```
Comment (cdaringe) [inline grunt/clean.js:1]: maybe it's my version of grunt, but the `<%= %>` templates were not putting the grunt config in the template context.  pulling hair out!

Comment (swashcap) [inline grunt/clean.js:1]: Huh, didn’t know that. Super frustrating, I imagine.

File: Gruntfile.js (hunk 1 above)
Comment (swashcap) [inline Gruntfile.js:8]: I’m a fan of keeping everything in the root _Gruntfile.js_ if it isn’t super long (<200 lines). But whatevs. Matches our other repos. `¯\_(GRUNTFILE)_/¯`

**Comment by swashcap** [comment]
Just tested this branch in MRN-Code/coins_core. Works great!

**Pull request merged**